### PR TITLE
Add download

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/annotation-cache/downloadvepcache/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -86,7 +42,6 @@ nextflow run annotation-cache/downloadvepcache -profile docker -params-file para
 with `params.yaml` containing:
 
 ```yaml
-input: './samplesheet.csv'
 outdir: './results/'
 genome: 'GRCh37'
```
Comment (maxulysse) [inline docs/usage.md:46]:
```suggestion
genome: 'GRCh38'
```

File: nextflow.config [hunk 2]
```diff
@@ -233,9 +218,6 @@ manifest {
     doi             = ''
 }
```
Comment (maxulysse) [inline nextflow.config:220]:
```suggestion

// Load DSL2 modules config
includeConfig 'modules/nf-core/ensemblvep/download/nextflow.config'

```

File: modules/nf-core/ensemblvep/download/ensemblvep-download.diff [hunk 3]
```diff
@@ -0,0 +1,59 @@
+Changes in module 'nf-core/ensemblvep/download'
+--- /dev/null
++++ modules/nf-core/ensemblvep/download/nextflow.config
+@@ -0,0 +1,20 @@
++/*
++~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++    annotation-cache/downloadvepcache ENSEMBLVEP_DOWNLOAD base config file
```
Comment (jfy133) [inline modules/nf-core/ensemblvep/download/ensemblvep-download.diff:7]: Why the process config within the module and not in modules.conf?

File: modules/nf-core/ensemblvep/download/ensemblvep-download.diff [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
++~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
++    A 'blank slate' config file, appropriate for general use on most high performance
++    compute environments. Assumes that all software is installed and available on
++    the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
++----------------------------------------------------------------------------------------
++*/
++
++process {
++    withName: 'ENSEMBLVEP_DOWNLOAD' {
```
Comment (jfy133) [inline modules/nf-core/ensemblvep/download/ensemblvep-download.diff:16]: Wait isn't this the same as the associated `nextflow.confg` alongisde the module files?

File: workflows/downloadvepcache.nf [hunk 5]
```diff
@@ -1,134 +1,17 @@
...
-    MULTIQC (
-        ch_multiqc_files.collect(),
-        ch_multiqc_config.toList(),
-        ch_multiqc_custom_config.toList(),
-        ch_multiqc_logo.toList()
-    )
-    multiqc_report = MULTIQC.out.report.toList()
+    emit:
+    ensemblvep_cache = ENSEMBLVEP_DOWNLOAD.out.cache.collect()  // channel: [ meta, cache ]
+    versions         = ENSEMBLVEP_DOWNLOAD.out.versions         // channel: [ versions.yml ]
```
Comment (jfy133) [inline workflows/downloadvepcache.nf:16]: Why keep this is you remove CUSTOM_DUMPSOFTWAREVERSIONS?

File: modules/nf-core/ensemblvep/download/ensemblvep-download.diff (hunk 3 above)
Comment (maxulysse) [reply modules/nf-core/ensemblvep/download/ensemblvep-download.diff:7]: not nf-core, so I'm shipping modules with config files to have all in one place

File: modules/nf-core/ensemblvep/download/ensemblvep-download.diff (hunk 4 above)
Comment (maxulysse) [reply modules/nf-core/ensemblvep/download/ensemblvep-download.diff:16]: yeah, that diff was created by the nf-core modules patch command, so no idea what happened there

File: workflows/downloadvepcache.nf (hunk 5 above)
Comment (maxulysse) [reply workflows/downloadvepcache.nf:16]: Might want to test for it in the future, but not interested in actually collecting the versions

**Pull request merged**